.claude/commands/tm/help.md

```
@@ -0,0 +1,90 @@
+Show help for Task Master commands.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)